test/structure/test_metal_atom.py

```diff
@@ -33,11 +25,13 @@ def test_constant_data():
     assert not ma.is_canonical()
     assert ma.rtype == chem.ResidueType.METAL
 
+
 def test_element():
-    stru = PDBParser().get_structure(f"{DATA_DIR}1NVG.pdb")
+    stru = PDBParser.get_structure(f"{DATA_DIR}1NVG.pdb")
```

Comment:
The idea of instantiate the object is to apply the interface check. Please change it back.

Reply:
sure

Reply:
Fixed.